docbuilder/src/main.rs

```diff
@@ -0,0 +1,215 @@
+// Cargo.toml dependencies:
```

Comment:
What's this guy doing?

Reply:
Cictrone wanted a script to gen docs from the pyi file. Im going to clean it up and get it implemented in build.rs. I can remove it for now if youre willing to merge this or you can wait until the docs auto update